packages/nsid/package.json

```diff
@@ -0,0 +1,19 @@
+{
+  "name": "@adxp/nsid",
```

Comment:
not saying this is wrong, but it does bring up the question for me of:
how granular do we want our packages to be?

I don't have a super strong opinion. But for instance, TIDs & AdxUris are both in `@adxp/common` so that may be a good home for NSIDs as well?

Reply:
Is it module madness to put them all in their own packages? I'm wondering if they should be -- at least nsid and adxuri